tough/src/io.rs

```diff
@@ -6,15 +6,15 @@ use ring::digest::{Context, SHA256};
 use std::io::{self, Read};
 use url::Url;
 
-pub(crate) struct DigestAdapter<T> {
```

Comment:
Is there a particular advantage to removing the type parameters on these implementation-detail types?